src/NServiceBus.Transport.Msmq/CheckMachineNameForComplianceWithDtcLimitation.cs

```diff
@@ -1,18 +1,18 @@
 namespace NServiceBus.Transport.Msmq
 {
+    using System;
     using System.Runtime.InteropServices;
     using System.Text;
-    using Logging;
 
-    class CheckMachineNameForComplianceWithDtcLimitation
+    static class CheckMachineNameForComplianceWithDtcLimitation
```

Comment:
I think class name be changed. I thought this was more of a MSMQ limitation than a DTC limitation. See John Breakwell's article. Thoughts @Particular/msmq-transport-maintainers ?